azerothcore/azerothcore-wotlk
# feat(DB/Creature): [Zone] Thousand Needles

Thousand Needles is now fully updated.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele ThousandNeedles and run / test around.
